datafusion/sqllogictest/test_files/aggregates_topk.slt

```diff
@@ -19,6 +19,12 @@
 # Setup test data table
 #######
 
+# Setting to map varchar to utf8view, to test PR https://github.com/apache/datafusion/pull/15152
+# Before the PR, the test case would not work because the Utf8View will not be supported by the TopK aggregation
```

Comment:
this change now loses coverage for utf8

Instead perhaps we can make a table with stringview?

Something like
```sql
CREATE TABLE traces_utf8view 
AS SELECT 
  arrow_cast(trace_id, 'Utf8View') as trace_id,
  timestamp,
  other
)
```

And then do an explain

```sql
explain select trace_id, MAX(timestamp) from traces group by trace_id order by MAX(timestamp) desc limit 4;
```

Reply:
Also it would be great to do the same fir `LargeUtf8`

Reply:
Good point, thank you @alamb.